customer-wallet/src/components/Drawer/index.js

```diff
@@ -26,6 +27,10 @@ const DrawerComponent = ({ toggleDrawer, drawerOpen, push }) => {
           <ListItemIcon><BitcoinIcon/></ListItemIcon>
           <ListItemText primary='Transactions' />
         </ListItem>
+        <ListItem button onClick={() => push('/pay')}>
+          <ListItemIcon><PaymentIcon/></ListItemIcon>
+          <ListItemText primary='Pay with Chlu' />
```

Comment:
"Pay with Chlu" sounds like the user should pay with Chlu tokens. 

If I understand correctly, this item links to the screen where the customer is directly paying with Bitcoin. There is no currency selection screen anymore, right? Are we sure that was taken out on purpose? Just want to be sure we are not missing something.

If it is skipping currency selection screen then we should call the item something like "Pay with Bitcoin"??

Reply:
Good point, it should be pay with bitcoin, I named it pay with chlu because I was thinking about the chlu protocol for reviews, but that's very confusing.

The currency selection got taken out and the UI component has been rewritten from scratch... this is because after we moved to material 1.0 I couldn't get the old one to work, it just froze my browser tab.

So after spending some time trying to get it to run I realized it would be quicker to just rewrite it. I kept the redux logic. The currency selection did nothing except open a modal that said only bitcoin is supported

Let me know what you think

Reply:
The currency selection is kind of important to communicate that we support payments through multiple cryptocurrencies. Lets try and make the effort to include it in this PR, otherwise I am afraid it might slip.

Reply:
Very good point about communicating that Chlu can work on multiple cryptocurrencies. Will try to figure out the best way to do that